CHANGELOG.md

```diff
@@ -203,6 +203,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - `Trainer.logged_metrics` now always contains scalar tensors, even when a Python scalar was logged ([#11270](https://github.com/PyTorchLightning/pytorch-lightning/pull/11270))
 
 
+- Improved the output of `SimpleProfiler` ([#11414](https://github.com/PyTorchLightning/pytorch-lightning/pull/11414))
```

Comment:
This would be a fix, right? Not an improvement.

Sorry to be that guy but this is going to need a test to avoid regressions :P

Some suggestions for that:
- Do not fit a full run, call a few hooks manually
- Mock the durations

Reply:
oh! I thought it was always like that.. never used with `Profiler(extended=False)`.

will try to add a test.

Reply:
Thanks @rohitgr7

Reply:
@carmocca mind check the test? is this something you had in mind?

Reply:
direct matching wasn't possible with windows, so made it a little dynamic.